### PR TITLE
Add short contribution section to primary README

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Adds a short "Contributing" section to the primary README with a link to the contributing guide and instructions on where to find good first issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,15 @@ Extensive documentation is available [here](http://community.optimism.io/docs/).
 * [`ops`](./ops): Contains Dockerfiles for containerizing each service involved in the protocol,
 as well as a docker-compose file for bringing up local testnets easily
 
+## Contributing
+
+Read through [CONTRIBUTING.md](./CONTRIBUTING.md) for a general overview of our contribution process.
+
+### Good First Issues
+You can find good first issues by filtering for the ["good first issue" tag on our issues page](https://github.com/ethereum-optimism/optimism/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or alternatively by taking a look at our [Good First Issues project board](https://github.com/orgs/ethereum-optimism/projects/23).
```
Comment (annieke) [inline README.md:45]: there are 7 open issues and 5 on the board, is that intentional? would be nice to have one as default truth

File: README.md [hunk 2]
```diff
@@ -37,6 +37,15 @@ Extensive documentation is available [here](http://community.optimism.io/docs/).
...
+If you'd like to tackle one of these issues, please leave a note on the issue.
```
Comment (annieke) [inline README.md:46]: nit: asking to also assign themself would make consistent practice too

**Review by annieke — COMMENTED** [review]
glad for this section, less maintenance burden if we automate the board to be exactly the filter

File: README.md (hunk 1 above)
Comment (tynes) [reply README.md:45]: One source of truth is better, I think we can deprecate the project board now that we have a monorepo?

Comment (smartcontracts) [reply README.md:45]: IMO both are necessary right now. We still have a few repos that are outside of the monorepo (the community hub is the main one) and my "good first issue bot" uses the project board.

Comment (tynes) [reply README.md:45]: Ok thats fine as long as you are willing to maintain both

**Review by tynes — APPROVED** [review]

**Pull request merged**